docs/dev-process/coding-guidelines.md

```diff
@@ -180,6 +180,21 @@ See [CODEOWNERS](../../.github/CODEOWNERS) for more information about the area e
   as it obscures the functional changes of the PR.
   A separate PR should be submitted for such refactoring without any functional changes.
 
+* Avoid initializing variables, fields and auto-implemented properties to zero or null because C# guarantees its are initialized to zero.
+
+* Follow Microsoft best practise guidances:
+  * [Framework Design Guidelines](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/index) - Naming, Design and Usage guidelines including:
+    * [Arrays](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/arrays)
+    * [Collections](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/guidelines-for-collections)
+    * [Exceptions](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/exceptions)
+  * [Best Practices for Developing World-Ready Applications](https://docs.microsoft.com/en-us/dotnet/standard/globalization-localization/best-practices-for-developing-world-ready-apps) - Unicode, Culture, Encoding and Localization.
+  * [Best Practices for System.Net Classes](https://docs.microsoft.com/en-us/dotnet/framework/network-programming/best-practices-for-system-net-classes)
```

Comment:
This one seems to focus on full .NET instead of .NET Core, so it mainly talks about things related to `WebRequest` and `WebResponse`.

Reply:
I added just for  WebRequest  and  WebResponse.
Remove?

Reply:
WebRequest and WebResponse we mostly deprecated, and HttpClient is the way to go. So I think we can remove this link.

Reply:
WebRequest and WebResponse we mostly deprecated, and HttpClient is the way to go. So I think we can remove this link.

Reply:
Removed.